src/plugins/intel_gpu/tests/unit/test_cases/crop_gpu_test.cpp

```diff
@@ -1334,12 +1331,36 @@ TEST_P(crop_gpu_dynamic, i32_in2x3x2x2_crop_offsets) {
         }
     }
 }
+using ConfigParams = std::tuple<impl_types, bool>;
+class crop_gpu_dynamic : public ::testing::TestWithParam<ConfigParams> {};
+TEST_P(crop_gpu_dynamic, i32_in2x3x2x2_crop_offsets) {
+    impl_types impl_type;
+    bool disable_usm;
+    std::tie(impl_type, disable_usm) = GetParam();
+    if (disable_usm) {
+#ifdef GPU_DEBUG_CONFIG
```

Comment:
`GPU_DEBUG_CONFIG` will not be set if `ENABLE_GPU_DEBUG_CAPS` is `OFF`, does CI enable it?

Reply:
hi river, It seems not, we can only manually trigger it. If so, is there any better way to test it？